app/controllers/queue_items_controller.rb

```diff
@@ -0,0 +1,39 @@
+class QueueItemsController < ApplicationController
+  before_filter :require_user
+
+  def index
+    @queue_items = current_user.queue_items
+  end
+
+  def create
+    vid = Video.find(params[:video_id])
```

Comment:
I don't recommend to abbreviate variable names. Make your code as much obvious as possible.